weekly-updates/weekly-update.2015-05-15.md

```diff
@@ -0,0 +1,27 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
```

Comment:
As a first step, we have moved from `iojs` organization to `nodejs` organization